tests/helper/tests/users.py

```diff
@@ -51,3 +55,24 @@ def users(client):
         if line != '':
             homes.append(line)
     assert len(homes) == 0, f"Found the following home directories: {homes}"
+
+
+def _has_user_sudo_cmd(client, user):
+    """ Check if user has any sudo permissions """
+    # Execute command on remote platform
+    cmd = f"sudo -l -U {user}"
+    out = utils.execute_remote_command(client, cmd)
+
+    # Write each line as output in list
+    output_lines = [] 
```

Comment:
why not simply calling it like this?
```
output_lines = out.split("\n")
```

Reply:
at this point, yes. But the loop will be needed later. tbh, we can reverse the order (as we expect the content within the last lines).